python/xorbits/_mars/dataframe/missing/fillna.py

```diff
@@ -164,9 +164,18 @@ def _execute_combine(cls, ctx, op):
         else:
             concat_df = pd.concat([valid_summary, input_data], axis=axis)
 
-        concat_df.fillna(
-            method=method, axis=axis, inplace=True, limit=limit, downcast=op.downcast
-        )
+        if is_pandas_2():
```

Comment:
The parameter `inplace` still works in pandas 2.0. Could you explain the reason of this change?

Reply:
In some cases, Pandas 2.0 will raise `NotImplementedError` with `inplace=True`.